### PR TITLE
fix: setup/teardown checkgroup scripts [sc-0]

### PR DESCRIPTION
**Pull request by nahuelon**
I hereby confirm that I followed the code guidelines found at [engineering guidelines](https://www.notion.so/checkly/Engineering-Guidelines-a3a165a203a04dc1a112f0e26b2f2d3f)

## Affected Components
* [x] CLI
* [ ] Create CLI
* [ ] Test
* [ ] Docs
* [ ] Examples
* [ ] Other

## Notes for the Reviewer
- fix the `localSetupScript` and `localTearDownScript` properties assignment for `CheckGroup`
- mark those properties as `deprecated`

> Resolves #[issue-number]

### CODE REVIEW

File: packages/cli/src/constructs/check-group.ts [hunk 1]
```diff
@@ -75,10 +75,12 @@ export interface CheckGroupProps {
   browserChecks?: BrowserCheckConfig,
   /**
    * A valid piece of Node.js code to run in the setup phase of an API check in this group.
-   */
+  * @deprecated use the "ApiCheck.setupScript" property instead
```
Comment (tnolet) [inline packages/cli/src/constructs/check-group.ts:78]:
```suggestion
  * @deprecated use the "ApiCheck.setupScript" property instead and use common JS/TS code composition to add group specific setup routines.
```

File: packages/cli/src/constructs/check-group.ts [hunk 2]
```diff
@@ -75,10 +75,12 @@ export interface CheckGroupProps {
...
+  */
   localSetupScript?: string
   /**
    * A valid piece of Node.js code to run in the teardown phase of an API check in this group.
+  * @deprecated use the "ApiCheck.tearDownScript" property instead
```
Comment (tnolet) [inline packages/cli/src/constructs/check-group.ts:83]:
```suggestion
  * @deprecated use the "ApiCheck.tearDownScript" property instead and use common JS/TS code composition to add group specific teardown routines.
```

**Review by tnolet — CHANGES_REQUESTED** [review]
just added some more helpful messaging.

**Review by umutuzgur — APPROVED** [review]

**Review by tnolet — DISMISSED** [review]
LGTM

**Review by tnolet — APPROVED** [review]
Lgtm

**Pull request merged**